Update Astronauts to Executive Leads and Change Election Process and …

pmoncada opened the pull request:
…Rights

- Update Astronauts to Executive Leads
- Introduce the idea of "primus inter pares" that manages budget decisions
- More definition about election process, yearly basis, or special election if someone doesn't finish their term, and include a "none" option.
- Two year term limit.
- Remove Exec Leads from Senate for separation of powers.

pmoncada committed:
Update Astronauts to Executive Leads and Change Election Process and Rights

- Update Astronauts to Executive Leads
- Introduce the idea of "primus inter pares"
- More definition about election process, yearly basis, or minimum of two leads, and include a "none" option.

pmoncada committed:
Fixed "an a" typo

ryand2d commented on MoonDAO/docs/Governance/Constitution.md:
```diff
@@ -110,7 +110,7 @@ MoonDAO’s Senate is composed of Senators. Proposals are first voted on by the
 ###### Election
 
-Each Quarter, five contributors (excluding Astronauts) are elected based on their performance in the previous quarter in order of retroactive rewards from the prior quarter. In addition, each active Project can select one Senator to serve on behalf of their Project for the quarter.
+Each Quarter, five contributors are elected based on their performance in the previous quarter in order of retroactive rewards from the prior quarter. In addition, each active Project can select one Senator to serve on behalf of their Project for the quarter.
```
should we also take advantage to clean up some things like repitition of previous quarter/prior quarter

pmoncada replied:
Good point, adding

ryand2d commented on MoonDAO/docs/Governance/Constitution.md:
```diff
@@ -123,48 +123,50 @@ Senators are active until the end of the quarter that they were appointed.
 
 ###### Election
-MoonDAO members may hold an election for a new Astronaut if there is a vacancy. Candidate Astronauts may be nominated by MoonDAO members and must have been part of the Senate for at least 180 days. Each nomination must go through the senate with a super majority vote. If there are multiple nominations that go through the Senate, then the DAO will vote among the candidates using $vMOONEY with Inst ...
+MoonDAO members may hold an election for a new Executive Lead on a yearly basis, or if there are less than two currently serving Executive Leads. Candidate Leads may be nominated by MoonDAO members and must be approved by the senate with a super majority vote. If there are multiple nominations that go through the Senate, then the DAO will vote among the candidates using $vMOONEY with Instant Runo ...
```
MoonDAO shall have up to three Executive Leads serving staggered three-year terms, with one regular election held each year to fill the seat of a term-limited or expiring Lead. If a seat becomes vacant before the completion of a term (e.g. due to resignation, removal, or death), the DAO may initiate a special election to fill the remainder of a vacant seat's term, within 60 days of the vacancy. MoonDAO members may vote for an eligible candidate or select the “none” option. If “none” wins a simple majority, the seat will remain vacant until the next eligible election. Only one regular Executive Lead election may occur per calendar year, unless a special election is required, if there are less than two currently serving Executive Leads. To maintain the staggered cycle, any special election winner will only serve the remainder of the original term they are replacing, not a full three-year term.

pmoncada replied:
I might try to break this up between election and term limits but keep the idea essentially the same

ryand2d commented on MoonDAO/docs/Governance/Constitution.md:
```diff
@@ -123,48 +123,50 @@ Senators are active until the end of the quarter that they were appointed.
 
 ###### Term Limits
-Astronauts serve for a three-year term. After three years, the DAO must re-vote with a super majority to confirm they stay as an Astronaut. They are eligible to remain an Astronaut for multiple terms. There can only be a maximum of three Astronauts at any given time.
+Executive Leads serve for a three-year term. After three years, the DAO must re-vote with a super majority to confirm they stay as an Executive Lead. They are eligible to remain a Lead for two terms. There can only be a maximum of three Executive Leads at any given time.
```
I feel like the second line should be clarified -- the executive lead 2nd term should be a general election -- not just a re appointment by super majority -- the person can or should go up against eligible candidates in a general election via RCV

pmoncada replied:
Agreed, clarified

ryand2d commented on MoonDAO/docs/Governance/Constitution.md:
```diff
@@ -123,48 +123,50 @@ Senators are active until the end of the quarter that they were appointed.
+Executive Leads may be removed through the creation of a Proposal that is approved by the Senate with a super majority vote, and the MoonDAO members with super majority vote over a five day voting period.
+
+###### Primus Inter Pares
+Among the Executive Leads there will be one person in charge of managing the yearly budget. The Primus Inter Pares, or Primus, is selected based on consensus of the Executive Leads. If no consensus is created, then they are selected by the Member House via a ranked choice vote.
```
is selected on a yearly basis based on consensus...

pmoncada replied:
Agreed - also added the deciding on the primus is within 14 days of the election ending

ryand2d commented on MoonDAO/docs/Governance/Constitution.md:
```diff
@@ -123,48 +123,50 @@ Senators are active until the end of the quarter that they were appointed.
+Treasury signers are appointed by the Executive Leads, and approved by the Senate and Member House with a super majority vote, and approval of a minimum 5 signers shall also be required. Executive Leads may also provide a list of people and allow for the Member house to choose by ranked choice voting, including a "none" option.
 
 ###### Term Limits
 There is no limit to the length of time a member can serve as a multi-signature member.
```
previously we said they would be referred to as "Signers" vs member here, which is confusing vs moondao member

pmoncada replied:
Can you clarify where you see member v.s. signer, I only see references to the "member house"

ryand2d commented on MoonDAO/docs/Governance/Constitution.md:
```diff
@@ -123,48 +123,50 @@ Senators are active until the end of the quarter that they were appointed.
 
 ###### Removal
-Treasury signers members can be removed by the Astronauts, and approved by the Senate and Member House with a super majority vote, and approval of a minimum 5 signers shall also be required.
+Treasury signers members may be removed by the Executive Leads, with approval by the Senate and Member House with a super majority vote, and approval of a minimum of 5 Treasury Signers.
```
signers members typo

pmoncada replied:
fixed, good catch

ryand2d commented on MoonDAO/docs/Governance/Constitution.md:
```diff
@@ -243,11 +245,11 @@ Moderators are guardians of the DAO, removing members that break the Code of Con
 ###### Rights
 
-1. Right to temporarily remove any member from MoonDAO official channels of communication for breaking the Code of Conduct. They must transparently provide a reason for the removal to the DAO. If the members have the right to vote in the MoonDAO senate, removal will require a review and approval by the Astronauts. If the member is a contributor on a Project, removal will require a review and appr ...
+1. Right to temporarily remove any member from MoonDAO official channels of communication for breaking the Code of Conduct. They must transparently provide a reason for the removal to the DAO. If the members have the right to vote in the MoonDAO senate, removal will require a review and approval by the Executive Leads. If the member is a contributor on a Project, removal will require a review and ...
```
did we also retire Rocketeer title vs Project Lead

pmoncada replied:
yes, changed

pmoncada committed:
Updated based on Ryan's Comments

- Clarified 60 day period for special elections
- Winner of special election serves remained of term of the seat they are replacing.
- Clarify that re-election occurs in an open election, not re-confirmation.
- More definition around how the 'primus' is decided.
- Fixed a couple typos or grammatical errors.
- Got rid of "Rocketeer" throughout the constitution.

pmoncada committed:
Fix instances of Treasury Signers referred to as "members"

pmoncada committed:
Removing accidental inclusion of 250k vmooney change
